mls-rs/Cargo.toml

```diff
@@ -67,7 +67,7 @@ spin = { version = "0.9.8", default-features = false, features = ["mutex", "spin
 maybe-async = { version = "0.2.10" }
 
 # Optional dependencies
-mls-rs-provider-sqlite = { path = "../mls-rs-provider-sqlite", version = "0.16.0", default-features = false, optional = true }
+mls-rs-provider-sqlite = { path = "../mls-rs-provider-sqlite", version = "0.17.0", default-features = false, optional = true }
```

Comment:
This dependency is re-exported by `mls-rs`, which makes it part of the public API of `mls-rs`, which implies this is also a breaking change for `mls-rs` and its version needs to be bumped accordingly (`0.48.0`).